# HG changeset patch
# User Rik <rik@octave.org>
# Date 1427689292 25200
#      Sun Mar 29 21:21:32 2015 -0700
# Node ID 59740a47f07055c256f57d09df03ed54e126df04
# Parent  d4d3bb493bcfbe9a15127130bfa97eb4e2a49061
it_IT.ts: Update Italian translation file.

* it_IT.ts: Updated translation from
Giorgio De Nunzio <giorgio.denunzio@unisalento.it>.

diff --git a/libgui/languages/it_IT.ts b/libgui/languages/it_IT.ts
--- a/libgui/languages/it_IT.ts
+++ b/libgui/languages/it_IT.ts
@@ -107,107 +107,107 @@
     <message>
         <location line="+3"/>
         <source>Open</source>
         <translation>Apri</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Annulla</translation>
     </message>
     <message>
         <location line="+0"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Annulla</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Close</source>
         <translation>&amp;Chiudi</translation>
     </message>
     <message>
         <location line="+0"/>
         <source>Close</source>
         <translation>Chiudi</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Apply</source>
-        <translation type="unfinished"></translation>
+        <translation>Applica</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Reset</source>
-        <translation type="unfinished"></translation>
+        <translation>Reset</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Help</source>
         <translation>Aiuto</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Don&apos;t Save</source>
-        <translation type="unfinished"></translation>
+        <translation>Non salvare</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Close without Saving</source>
-        <translation type="unfinished"></translation>
+        <translation>Chiudi senza salvare</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Discard</source>
-        <translation type="unfinished"></translation>
+        <translation>Elimina</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Yes</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;S&igrave;</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Yes to &amp;All</source>
-        <translation type="unfinished"></translation>
+        <translation>S&igrave; a &amp;tutto</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;No</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;No</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>N&amp;o to All</source>
-        <translation type="unfinished"></translation>
+        <translation>N&amp;o a tutto</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Save All</source>
-        <translation type="unfinished"></translation>
+        <translation>Salva tutto</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Abort</source>
-        <translation type="unfinished"></translation>
+        <translation>Annulla</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Retry</source>
-        <translation type="unfinished"></translation>
+        <translation>Riprova</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Ignore</source>
-        <translation type="unfinished"></translation>
+        <translation>Ignora</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Restore Defaults</source>
-        <translation type="unfinished"></translation>
+        <translation>Ripristina defaults</translation>
     </message>
 </context>
 <context>
     <name>QErrorMessage</name>
     <message>
         <location filename="../../../octave-qt/qerrormessage.cpp" line="+208"/>
         <source>Debug Message:</source>
         <translation>Messaggio di Debug:</translation>
@@ -1827,22 +1827,22 @@ Si desidera sovrascriverlo?</translation
     <message>
         <location filename="../graphics/MouseModeActionGroup.cc" line="+40"/>
         <source>Rotate</source>
         <translation>Ruota</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Zoom In</source>
-        <translation type="unfinished">Ingrandisci</translation>
+        <translation>Ingrandisci</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Zoom Out</source>
-        <translation type="unfinished">Riduci</translation>
+        <translation>Riduci</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Pan</source>
         <translation>Panoramica</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -2491,17 +2491,17 @@ Si desidera scegliere un altro nome?</tr
     <message>
         <location line="-783"/>
         <source>Show Octave directory</source>
         <translation>Mostra la cartella di Octave</translation>
     </message>
     <message>
         <location line="-4"/>
         <source>One directory up</source>
-        <translation type="unfinished">Livello precedente</translation>
+        <translation>Livello precedente</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Go to current Octave directory</source>
         <translation>Vai alla directory corrente di Octave</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -3596,17 +3596,17 @@ L&apos;inerfaccia grafica verrà chiusa.</translation>
         <location filename="../src/settings-dialog.ui" line="+23"/>
         <source>Settings</source>
         <translation>Impostazioni</translation>
     </message>
     <message>
         <location line="+19"/>
         <location line="+468"/>
         <source>General</source>
-        <translation>Genrale</translation>
+        <translation>Generale</translation>
     </message>
     <message>
         <location line="-338"/>
         <source>Octave logo only</source>
         <translation>Logo di Octave solamente</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -3908,32 +3908,32 @@ L&apos;inerfaccia grafica verrà chiusa.</translation>
     <message>
         <location line="-1715"/>
         <source>Interface</source>
         <translation>Interfaccia</translation>
     </message>
     <message>
         <location line="+61"/>
         <source>Small</source>
-        <translation type="unfinished"></translation>
+        <translation>Piccolo</translation>
     </message>
     <message>
         <location line="+7"/>
         <source> Normal</source>
-        <translation type="unfinished"></translation>
+        <translation>Normale</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Large</source>
-        <translation type="unfinished"></translation>
+        <translation>Grande</translation>
     </message>
     <message>
         <location line="+200"/>
         <source>3D</source>
-        <translation type="unfinished"></translation>
+        <translation>3D</translation>
     </message>
     <message>
         <location line="+54"/>
         <source>Octave Startup</source>
         <translation>Avvio di Octave</translation>
     </message>
     <message>
         <location line="+41"/>
@@ -3955,17 +3955,17 @@ L&apos;inerfaccia grafica verrà chiusa.</translation>
     <message>
         <location line="+64"/>
         <source>max.</source>
         <translation>max.</translation>
     </message>
     <message>
         <location line="+221"/>
         <source>Indentation</source>
-        <translation type="unfinished"></translation>
+        <translation>Indentazione</translation>
     </message>
     <message>
         <location line="+256"/>
         <source>With Octave builtins</source>
         <translation>Con built-in di Octave</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -3975,17 +3975,17 @@ L&apos;inerfaccia grafica verrà chiusa.</translation>
     <message>
         <location line="+64"/>
         <source>Show completion list automatically</source>
         <translation>Mostra lista di completamento automaticamente</translation>
     </message>
     <message>
         <location line="+31"/>
         <source>File handling</source>
-        <translation type="unfinished"></translation>
+        <translation>Gestione file</translation>
     </message>
     <message>
         <location line="+40"/>
         <source>Reload externally changed files without prompt</source>
         <translation>Ricarica file modificati esternamente senza chiedere</translation>
     </message>
     <message>
         <location line="+53"/>
